pkg/scripts/install.sh

```diff
@@ -53,14 +53,14 @@ EOF
 process_arguments() {
   # set the default value for config file if is not provided
   if [ -z "${CONFIG_FILE}" ]; then
-    CONFIG_FILE="${SCRIPT_PATH}/../../config.tfvars"
```

Comment:
All reference to files are based on the location of scripts - This is because of prevent to obligate user from running  the scripts from the root of repo. We better to keep this to be consistent with this rule.

Reply:
I think users should run the scripts from the root, I would almost argue the `install.sh` and `uninstall.sh` scripts should be located in the root folder.

Yes, we should make it more flexible so you can call it from anywhere, but that doesn't work even in the current implementation.

Reply:
The bug has fixed in PR#69 and this feature is working fine (running from anywhere). It means we can safely move install and uninstall scripts to root and it expected to work with no further required change